source/index.html.md.erb

```diff
@@ -184,6 +184,34 @@ Parameter | Description
 --------- | -----------
 recruitment_cycle | 4-character year (e.g. 2019 for 2019/20 courses)
 
+### Get all changed courses
+
+This endpoint retrieves all courses changed since midnight UTC of the specified day.
```

Comment:
I don't see how we specify a day. Do you mean the current day the request is made?